App.js

```diff
@@ -0,0 +1,72 @@
+import TodoInput from "./src/components/TodoInput.js"
+import TodoList from "./src/components/TodoList.js"
+import TodoListRender from "./src/components/TodoListRender.js"
+import TodoCount from "./src/components/TodoCount.js"
+import TodoFilter from "./src/components/TodoFilter.js"
+
+const $todoInput = document.querySelector("#new-todo-title");
+const $todoList = document.querySelector("#todo-list");
+const $todoCount = document.querySelector(".todo-count");
```

Comment:
todoCount 범위를 좁히면 좋을 것 같아요!
```suggestion
const $todoCount = document.querySelector("strong");
```
또는 더 엄격하게
```suggestion
const $todoCount = document.querySelector(".todo-count strong");
```